recipes/gdcm/all/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.9.2)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(NOT "${CMAKE_CXX_STANDARD}")
```

Comment:
Maybe?

```suggestion
if(NOT DEFINED CMAKE_CXX_STANDARD)
```

Reply:
it was copied from some other recipe but NOT DEFINED definitely looks better